### DISCUSSION TITLE
Does Official API deduct credits

### DISCUSSION THREAD

**Discussion by ahmedivy** [acheong08/ChatGPT#498] [Q&A] [answered]
Does it deduct credits from our openai account?

**Comment by acheong08** [comment]
Yes

**Reply by ahmedivy** [reply]
Almost same rates per tokens like GPT-3 or significantly more or less?

**Reply by acheong08** [reply]
Same as davinci (GPT-3)

**Reply by ahmedivy** [reply]
Ok. ThankYou

**Comment by acheong08** [comment, marked as answer]
@ahmedivy It was just discovered that if you use the model with an account without credit, it's free

**Reply by acheong08** [reply]
They might patch it at some point but for now it is not rate limited and completely free

**Reply by ahmedivy** [reply]
oh! nice
